chsdi/templates/wmtscapabilities.mako

```diff
@@ -94,7 +95,7 @@ else:
                 % if epsg == '21781':
                     <TileMatrixSet>${str(layer.tile_matrix_set_id).split(',')[0]}_${str(layer.zoomlevel_max)|validate_tilematrixset}</TileMatrixSet>
                 % else:
-                    <TileMatrixSet>${epsg}</TileMatrixSet>
+                    <TileMatrixSet>${epsg}_${str(layer.zoomlevel_max)}</TileMatrixSet>
```

Comment:
How is it gona work with `EPSG:3852`. I think you need to adapt these templates as well: (identifier is `3857` and not `EPSG:3857_maxzoomLevel`) Right?

https://github.com/geoadmin/mf-chsdi3/blob/mom_test_axis_order/chsdi/templates/TileMatrixSet_3857.mako